pkg/collectors/devices/ptp_devices.go

```diff
@@ -12,77 +13,173 @@ import (
 )
 
 type PTPDeviceInfo struct {
-	VendorID string `json:"vendorId"`
-	DeviceID string `json:"deviceInfo"`
-	GNSSDev  string `json:"GNSSDev"` //nolint:tagliatelle // Because GNSS is an ancronym
+	Timestamp string `json:"date"`
```

Comment:
Can we change the `json` tag to something more descriptive? I assume it's actually quite a bit more precise than a date.

Reply:
its just the iso datetime with nano seconds. could put timestamp instead.